### PR TITLE
feat(theme/i18n): allow customizing sponsor link's text

### LINKED ISSUE #3273: sponsor_title to VPTeamMembers
### Is your feature request related to a problem? Please describe.

Add please cuspomize sponsor_title to VPTeamMembers

### Describe the solution you'd like

Translate "sponsor" in the buttom, to "Поддержать"

### Describe alternatives you've considered

_No response_

### Additional context

_No response_

### Validations

- [X] Follow our [Code of Conduct](https://vuejs.org/about/coc.html)
- [X] Read the [docs](https://vitepress.dev).
- [X] Read the [Contributing Guidelines](https://github.com/vuejs/vitepress/blob/main/.github/contributing.md).
- [X] Check that there isn't already an issue that asks for the same feature to avoid creating a duplicate.

### PR DESCRIPTION
**Pull request by pokedim13**
closes #3273

Add sponsor_text 
Снимок экрана от 2023-11-30 06-17-47

### CODE REVIEW

**Comment by pokedim13** [comment]
https://github.com/vuejs/vitepress/issues/3273

File: types/default-theme.d.ts [hunk 1]
```diff
@@ -345,6 +345,7 @@ export namespace DefaultTheme {
     desc?: string
     links?: SocialLink[]
     sponsor?: string
+    sponsor_text?: string
```
Comment (kiaking) [inline types/default-theme.d.ts:348]: This should be:

```suggestion
    sponsorText?: string
```

File: src/client/theme-default/components/VPTeamMembersItem.vue [hunk 2]
```diff
@@ -47,7 +47,7 @@ withDefaults(defineProps<Props>(), {
     </div>
     <div v-if="member.sponsor" class="sp">
       <VPLink class="sp-link" :href="member.sponsor" no-icon>
-        <VPIconHeart class="sp-icon" /> Sponsor
+        <VPIconHeart class="sp-icon" /> {{ member.sponsor_text || "Sponsor" }}
```
Comment (brc-dd) [inline src/client/theme-default/components/VPTeamMembersItem.vue:50]:
```suggestion
        <VPIconHeart class="sp-icon" /> {{ member.actionText || "Sponsor" }}
```

File: types/default-theme.d.ts (hunk 1 above)
Comment (brc-dd) [reply types/default-theme.d.ts:348]:
```suggestion
    actionText?: string
```

**Review by brc-dd — APPROVED** [review]

**Pull request merged**